Dockerfile

```diff
@@ -89,6 +91,8 @@ COPY --from=artery-build /artery/lib /artery/lib
 COPY --from=artery-build /artery/share/ned /artery/share/ned
 ENV SUMO_HOME /sumo/share/sumo
 ENV PATH /sumo/bin:/omnetpp/bin:$PATH
+RUN ln -s /usr/bin/python3.9 /usr/bin/python3 && \
```

Comment:
I think the *python3* package sets this up, doesn't it?

Reply:
It did not for me unfortunately, the `python3` package only provides the `python3.9` binary.

Reply:
Okay, got it. `ln -s /usr/bin/python3 /usr/bin/python` is sufficient, though.